qa/tasks/mon_thrash.py

```diff
@@ -220,6 +221,18 @@ def max_killable(self):
             return m
 
     def do_thrash(self):
+        """
+        _do_thrash() wrapper.
+        """
+        try:
+            self._do_thrash()
+        except Exception as e:
+            # See _run exception comment for MDSThrasher
+            self.e = e
+            self.logger.exception("exception:")
+            # allow successful completion so gevent doesn't see an exception...
```

Comment:
And note that the (soon to be merged) DaemonWatchdog will observe the error and tear down the test.

Reply:
@batrick Fixed. Please check.